Update fleet-server-configuration.md adding allow_private_network (#50422)
fleetdm/fleet: docs/Configuration/fleet-server-configuration.md

    websockets_allow_unsafe_origin: true
  ```

### server_allow_private_network_integrations

> Allows Fleet's HTTP client to make outbound requests to RFC 1918 and other private network addresses. Enable this if an integration Fleet needs to reach over HTTP. Such as an SSO/IdP, EJBCA, Jira, or SCEP server, or an `HTTP_PROXY`/`HTTPS_PROXY` hosted on a private network.

Reviewer: ```suggestion
Allows Fleet's HTTP client to make outbound requests to RFC 1918 and other private network addresses. Enable this if  Fleet needs to reach an integration over HTTP. (Examples include SSO/IdP, EJBCA, Jira, or SCEP server, or an `HTTP_PROXY`/`HTTPS_PROXY` hosted on a private network.)
```
